dockyard/api/config.py

```diff
@@ -0,0 +1,24 @@
+# Pecan Application Configurations
+app = {
+    'root': 'dockyard.controllers.root.RootController',
+    'modules': ['dockyard', 'dockyard.api'],
+    'static_root': '%(confdir)s/public',
+    'template_path': '%(confdir)s/dockyard/templates',
+    'debug': True,
```

Comment:
It is ok right now but in production environment, it should not be in the debug mode.